### ISSUE TITLE
Search results are cleared when reloading VSCode

### ISSUE THREAD

**Issue by jespertheend** [microsoft/vscode#145350]
Does this issue occur when all extensions are disabled?: Yes

Version: 1.65.0-insider (Universal)
Commit: b5205cc8eb4fbaa726835538cd82372cc0222d43
Date: 2022-03-02T03:14:48.052Z
Electron: 13.5.2
Chromium: 91.0.4472.164
Node.js: 14.16.0
V8: 9.1.269.39-electron.0
OS: Darwin x64 21.3.0

**Steps to Reproduce**:

1. Press `cmd`/`ctrl` `shift` `F` and search for something in your project
2. Restart VSCode, or open the command menu and choose `Developer: Reload Window`

** Expected result **
The search result is remembered, or the search query is performed again upon opening the search panel.

** Actual result **
The search panel is empty, acting as if the search field is still empty, only after modifying the search query will it start searching again.

https://user-images.githubusercontent.com/2737650/158883459-a451e733-ce5a-4bd1-9e55-a7634b1a1430.mp4

**Comment by roblourens** [comment]
I definitely don't want to trigger the search again when the window opens - searches are CPU intense and could slow down vscode startup.

I don't think I want to persist the results either - it seems just as likely that they will be out of date the next time you open that folder, so I will want to rerun the search anyway.

The query is saved, you can press the arrow keys to navigate history. Why do you want the results persisted?

**Comment by jespertheend** [comment]
@roblourens Yeah if you close the project and open it again a few days later I don’t think it makes sense for the results to persist. But in my specific case just now I was specifically reloading the window while I was in the middle of a task that needed a search.

The results being empty in this case wasn’t a big annoyance for me or anything. But I was confused as to why all the results were gone now. Specifically I was working on removing all occurrences of the query, so I wasn’t sure if I was already done with my task or if the query hadn’t been triggered.

I don’t think performance should be an issue as the search panel isn’t open by default after reloading. So you could trigger the query once the panel is opened. That way it doesn’t have to interfere with startup times.

**Comment by roblourens** [comment]
I think I disagree in this case, there is some transient state that will be lost when the window is reloaded. I don't want to optimize for reloading the window in the middle of one task.

**Comment by jespertheend** [comment]
Fair enough, I can live with it either way. I think it was mostly the text still being in the text field that made it seem like something was broken. But I don't mind the current behavior.
